librfn/libopencm3/console_cdcacm.c

```diff
@@ -291,9 +291,9 @@ static int usb_fibre(fibre_t *fibre)
 }
 static fibre_t usb_task = FIBRE_VAR_INIT(usb_fibre);
 
-static int cdcacm_control_request(usbd_device *dev,
+static enum usbd_request_return_codes cdcacm_control_request(usbd_device *dev,
```

Comment:
Can we make the patch description more specific (and to make it easier for projects that use librfn to chase down problems due to this). Something like:

Since 54b117c5a576 ("usb: Use enumerated return codes") libopencm3 updated the callback to return an enum, let's use it here as well.